src/calculateRentalCost.js

```diff
@@ -30,18 +30,21 @@
 function calculateRentalCost(days) {
   // write code here
   const rentalPrice = 40;
+  const rentalAmount = days * rentalPrice;
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
   let total = 0;
 
   if (days < 3) {
-    total = days * rentalPrice;
+    total = rentalAmount;
```

Comment:
Your code works fine but you could still improve it by returning `rentalAmount` here and the same thing in the following if statements. However you will need to mind the sequence of your if statements.